wiki/BanchoBot/zh.md

```diff
@@ -5,7 +5,7 @@ outdated: true
 
 ![BanchoBot 玩家信息](img/BanchoBot.jpg "BanchoBot 玩家信息")
 BanchoBot 是由 **Echo49** 编写的一个可爱的机器人。它能在聊天室中根据用户下达的指令来给予反馈，从而协助玩家例如显示特定信息以及发布与游戏相关的内容。
- 
+ 
```

Comment:
delete this space i think

Reply:
I kept all incorrect spacings as-is because I'm splitting these tasks across many PRs. don't worry, I'm keeping track :^)